packages/web-pixels-extension/src/types/EventBus.ts

```diff
@@ -1,38 +1,3 @@
-import type {PixelEvents, CustomEvent, EventType} from './PixelEvents';
```

Comment:
Do we know why all these changes are here?

Reply:
Not sure unfortunately, they came from running `yarn schema:all`. @james-a-c @sandor-shopify do you guys have any ideas?

Reply:
I'm not sure either. I'm seeing these same changes on my branch when running yarn schema:all